docs.html

```diff
@@ -3,22 +3,16 @@
 title:       "JBossWS Documentation"
 link:        docs
 ---
-<h3>Documentation</h3>
-<p><em><strong>Main SNAPSHOT documentation</strong></em></p>
-<p><a href="{{site.baseurl}}/documentation/latest/JBossWS-CXF">6.0.1-SNAPSHOT</a> </p>
 
-<p><em><strong>Release documentation</strong></em></p>
-<p><a href="{{site.baseurl}}/documentation/6.0.0.Final/JBossWS-CXF">6.0.0.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.5.0.Final/">5.5.0.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.4.4.Final/">5.4.4.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.3.0.Final/">5.3.0.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.2.4.Final/">5.2.4.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.1.9.Final/">5.1.9.Final</a> </p>
 
-<!--
-<p>The <a href="https://docs.jboss.org/author/display/JBWS">documentation for the latest JBossWS
-    series</a> is available and continuously updated.</p>
-<p>Starting from JBossWS 5.0.0.Final, the release documentation is exported at
-    <a href="https://docs.jboss.org/jbossws/">https://docs.jboss.org/jbossws/</a>.</p>
-<p>&nbsp;</p>
--->

```

Comment:
Why are these documentation links removed ?

Reply:
Link does not exist